src/components/user/UserDetailsComponent.jsx

```diff
@@ -5,7 +5,19 @@ class UserDetailsComponent extends Component {
     render() {
         return (
             <div>
-               <h5>{this.props.data.name}</h5>
+                <div>
+                    <h5>{this.props.data.name}</h5>
+                </div>
+                <div>
+                    {this.props.data.teams.forEach(team => {
```

Comment:
Use `.map` instead of `.forEach` for cleaner code. `.map` is dynamically updated.